clients/js/src/generated/instructions/index.ts

```diff
@@ -6,13 +6,20 @@
  * @see https://github.com/metaplex-foundation/kinobi
  */
 
+export * from './addCanopy';
+export * from './appendCanopyNodes';
 export * from './burn';
 export * from './cancelRedeem';
 export * from './createTreeConfig';
 export * from './decompressV1';
 export * from './delegate';
+export * from './finalizeTreeWithRoot';
+export * from './finalizeTreeWithRootAndCollection';
+export * from './initPreparedTreeWithRoot';
 export * from './mintToCollectionV1';
 export * from './mintV1';
+export * from './prepareBatchMerkleTree';
+export * from './prepareTree';
```

Comment:
I think these spl-account-compression instructions such as `appendCanopyNodes` do not need to be included in our JS and Rust clients. They are added by default and can be removed in kinobi. See kinobi.cjs where other spl-account-compression instructions such as `append` and `closeEmptyTree` are marked for deletion.